lib/browser/remote/server.ts

```diff
@@ -321,7 +321,7 @@ const unwrapArgs = function (sender: electron.WebContents, frameId: number, cont
 
 const isRemoteModuleEnabledImpl = function (contents: electron.WebContents) {
   const webPreferences = (contents as any).getLastWebPreferences() || {}
-  return !!webPreferences.enableRemoteModule
+  return webPreferences.enableRemoteModule != null ? !!webPreferences.enableRemoteModule : true
```

Comment:
I think the reason why the default was `false` is that devtools webContents don’t have webPreferences and the remote module should be disabled there

Reply:
Hm, do you have suggestions for how we can detect devtools webcontents but also know whether remote module was explicitly enabled?

Reply:
I can do this in a follow-up PR if I don’t forget